Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Components/Pages/PagesControllerImpl.cs

```diff
@@ -1217,9 +1217,29 @@ private static string GetExternalUrlRedirection(string url)
             if (url == null)
             {
                 return null;
-            }
-
-            return url.ToLower() == "http://" ? "" : Globals.AddHTTP(url);
+            }
+
+            if (url.ToLower() == "http://")
+            {
+                return string.Empty;
+            }
+
+            if (url.StartsWith("//"))
+            {
+                return Globals.AddHTTP(url.Remove(0, 2));
+            }
+
+            if (url.ToLower().StartsWith("http"))

```

Comment:
```suggestion
            if (url.IndexOf("://") != -1)

```

And this will cover other protocols like `ftp://someserver.com/folder/file.zip`

Reply:
Excellent point @valadas - I am testing this change now.